tests/wpt/metadata/XMLHttpRequest/data-uri.htm.ini

```diff
@@ -1,6 +1,5 @@
 [data-uri.htm]
   type: testharness
-
```

Comment:
Personally, I think it's fine to leave it in. Ideally it'd be in a separate PR, but not a big deal at all. I'll let @KiChjang decide though